src/_product-updates/2024-08-10-feedbacks-gathering.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Documentation feedback 
+tags: [New]
+image: /images/changelog/feedback.png
```

Comment:
I'd change the file name, as we could want to use "feedback.png" for a bigger update in the future :)
What about "feedback-button.png"?

Reply:
It's the name of my screenshot of the actual button, but I guess we can have the thumbnail be "feedback-button", and the screenshot "feedback-button-mockup"??

Reply:
Will be pushed with the next commit 🙏